### ISSUE TITLE
TravisCI false positive check

### ISSUE THREAD

**Issue by Capstan** [java-json-tools/json-schema-validator#308]
https://travis-ci.org/java-json-tools/json-schema-validator/builds/578753562 was triggered on pr #292 and a fatal error still resulted in a positive result. This should track reporting the actual error and also fixing travis to not call this a "pass".

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by net.bytebuddy.dynamic.loading.ClassInjector$UsingReflection$Dispatcher$Resolved (file:/home/travis/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.5.5/8557b6465cea17f3769678235e77d5cb076c1170/byte-buddy-1.5.5.jar) to method java.lang.ClassLoader.findLoadedClass(java.lang.String)
WARNING: Please consider reporting this to the maintainers of net.bytebuddy.dynamic.loading.ClassInjector$UsingReflection$Dispatcher$Resolved
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by Capstan** [comment]
Ah, never mind; it's warnings not errors.
